python/packages/autogen-core/src/autogen_core/components/models/__init__.py

```diff
@@ -1,23 +1,19 @@
 from ._model_client import ChatCompletionClient, ModelCapabilities
-from ._openai_client import (
-    AzureOpenAIChatCompletionClient,
-    OpenAIChatCompletionClient,
-)
 from ._types import (
     AssistantMessage,
+    ChatCompletionTokenLogprob,
     CreateResult,
     FinishReasons,
     FunctionExecutionResult,
     FunctionExecutionResultMessage,
     LLMMessage,
     RequestUsage,
     SystemMessage,
+    TopLogprob,
     UserMessage,
 )
 
 __all__ = [
-    "AzureOpenAIChatCompletionClient",
```

Comment:
We will also need to show deprecation warnings when user attempt to import from `autogen_core.components.models`

Reply:
Massive +1 to this

Reply:
Done. Couldn't remove the code and make a conditional import due to circular dependencies but the warning was added